packages/runtime-core/src/vnode.ts

```diff
@@ -20,6 +20,8 @@ export const Comment = __DEV__ ? Symbol('Empty') : Symbol()
 export const Portal = __DEV__ ? Symbol('Portal') : Symbol()
 export const Suspense = __DEV__ ? Symbol('Suspense') : Symbol()
 
+export const vnodeSymbol = Symbol()
```

Comment:
Should this be named in dev as other symbols (`__DEV__ ? Symbol('isVNode') : Symbol()`)?

Reply:
I think, yes.

Reply:
@znck done